### ISSUE TITLE
Text paste exception

### ISSUE THREAD

**Issue by Compusomnia** [PowerShell/PSReadLine#1716]
```
PS D:\audio\generated>
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 2 Keys:
 ` Shift+Insert

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -1.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Paste(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------

```

**Comment by vexx32** [comment]
What version of PSReadLine are you using & what version of PowerShell? (There should be example code for retrieving this data in the issue templates iirc)

**Comment by Compusomnia** [comment]
PS version: 5.1.18362.752
PSReadline version: 2.0.0-beta2
os: 10.0.18362.1 (WinBuild.160101.0800)
PS file version: 10.0.18362.1 (WinBuild.160101.0800)
HostName: ConsoleHost
BufferWidth: 105
BufferHeight: 5000

I must have submitted an issue through a means where the version code didn't show up. Maybe it would be more visible in the wiki or readme as well?

**Comment by vexx32** [comment]
Yeah, agreed. It should be somewhere else as well, though not sure exactly where would be appropriate for this repo. @daxian-dbw  might have some ideas.

Beta2... hmm, that one's fairly old. Can you try using `Update-Module PSReadLine` (or `Install-Module PSReadLine -Force` if that fails) to install a more recent version?

This might have already been fixed. 🤔

**Comment by Compusomnia** [comment]
```
PS D:\audio\generated> Update-Module PSReadLine

NuGet provider is required to continue
PowerShellGet requires NuGet provider version '2.8.5.201' or newer to interact with NuGet-based
repositories. The NuGet provider must be available in 'C:\Program
Files\PackageManagement\ProviderAssemblies' or
'C:\Users\Tristan\AppData\Local\PackageManagement\ProviderAssemblies'. You can also install the NuGet
provider by running 'Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force'. Do you want
PowerShellGet to install and import the NuGet provider now?
[Y] Yes  [N] No  [S] Suspend  [?] Help (default is "Y"):
Update-Module : Module 'PSReadLine' was not installed by using Install-Module, so it cannot be updated.
At line:1 char:1
+ Update-Module PSReadLine
+ ~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (PSReadLine:String) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : ModuleNotInstalledUsingInstallModuleCmdlet,Update-Module

PS D:\audio\generated>
```

I had to do the second command through admin rights and now when I start up a PowerShell instance, I get:

```
Cannot load PSReadline module.  Console is running without PSReadline.
```

I don't know what I did wrong.

**Comment by vexx32** [comment]
Try doing an explicit `Import-Module PSReadLine` and see what error it's hitting. 🙂

**Comment by Compusomnia** [comment]
```
Cannot load PSReadline module.  Console is running without PSReadline.
PS D:\audio\generated> Import-Module PSReadLine
Import-Module : File C:\Program Files\WindowsPowerShell\Modules\PSReadLine\2.0.3\PSReadLine.psm1 cannot
be loaded because running scripts is disabled on this system. For more information, see
about_Execution_Policies at https:/go.microsoft.com/fwlink/?LinkID=135170.
At line:1 char:1
+ Import-Module PSReadLine
+ ~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : SecurityError: (:) [Import-Module], PSSecurityException
    + FullyQualifiedErrorId : UnauthorizedAccess,Microsoft.PowerShell.Commands.ImportModuleCommand
```

I don't recall explicitly disabling anything.

**Comment by vexx32** [comment]
Ah, of course, forgot that the module format changed slightly across the versions. You can use `Set-ExecutionPolicy AllSigned` or `Set-ExecutionPolicy RemoteSigned` to sort that out. 🙂

**Comment by daxian-dbw** [comment]
This issue was fixed (see #1306). Please upgrade to the 2.0.4 version of PSReadLine from PowerShellGallery. See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. Please let us know if you run into the same issue with the latest version.
